pages/dkp/konvoy/2.1/choose-infrastructure/azure/advanced/new/index.md

```diff
@@ -21,30 +40,34 @@ enterprise: false
     CLUSTER_NAME=my-azure-cluster
     ```
 
-## Tips and Tricks
+### Naming Tips and Tricks
 
-1.  To create a cluster name that's unique, use the following command:
+1.  To create a cluster name that is unique, use the following command:
 
     ```sh
     CLUSTER_NAME=$(whoami)-azure-cluster-$(LC_CTYPE=C tr -dc 'a-z0-9' </dev/urandom | fold -w 5 | head -n1)
     echo $CLUSTER_NAME
     ```
 
+    The output appears similar to:
+
     ```text
     hunter-azure-cluster-pf4a3
     ```
 
-    This will create a unique name every time you run it, so use it with forethought.
+    This command creates a unique name every time you run it, so use it carefully.
 
 1.  Set the environment variable to the name you assigned this cluster:
 
     ```sh
     CLUSTER_NAME=my-azure-cluster
     ```
 
-## Create a new Azure Kubernetes cluster
+## Create new Azure Kubernetes cluster objects
 
-1.  Generate the Kubernetes cluster objects:
+This procedure uses the `--dry-run` and `--output-yaml` flags together to create basic Azure Kubernetes cluster objects in a YAML file. This approach allows you to examine the YAML objects before creating the actual Azure Kubernetes cluster itself. For example, you may need to specify an HTTP proxy for your control plane and worker nodes. You could also configure the cluster to use your existing network infrastructure. If you need either of those additional configurations, skip to the relevant procedures _without_ executing this one.
```

Comment:
The last sentence here is incorrect because you should only skip this step if you need an http proxy.   The "Existing network" section does not instruct you to create the objects, so if you skip from here to that section, you won't have a configuration to inspect and edit.   Also, I find the 'choose your own adventure' format here to be kind of verbose and confusing.   There's really just two different starting points here (either with or without a proxy) and then an optional step where you can mess with the network config if you need to use existing network infra)...

```suggestion
This procedure uses the `--dry-run` and `--output-yaml` flags together to create basic Azure Kubernetes cluster objects in a YAML file. This approach allows you to examine the YAML objects before creating the actual Azure Kubernetes cluster itself.   

When creating the basic Azure Kubernetes cluster objects, you need to first consider if you need to use an HTTP proxy.   If you do so, then you will need to do some additional configuration when creating the cluster objects.   Consult the "Configure the control plane and workers to use an HTTP Proxy" section for more details. 
```

Reply:
Resolved as noted in my local branch.

Reply:
Ideally, we would break up these several headings into several different topics and cross-link between them as needed to get the right selections into the cluster. I am loathe to nest too many levels down in the navigation hierarchy, lest headings get lost from display and search. They are still somewhat "lost" here, too, but at least they are in one place, and we have the internal links for this page. Not the best solution, but enough perhaps with the release just 2 days off.